lms/services/blackboard_api/client.py

```diff
@@ -27,20 +27,28 @@ def get_token(self, authorization_code):
         """
         self._api.get_token(authorization_code)
 
-    def list_files(self, course_id):
-        """Return the list of files in the given course."""
+    def list_files(self, course_id, folder_id=None):
```

Comment:
`list_files` I think is fine as a method name (list_resources doesn't mean much) but maybe include the fact that can return folders in the docstring.

Reply:
https://github.com/hypothesis/lms/pull/2950/commits/e55f288cef33d2da763325636c98be9dc9e7292c